packages/core/src/dynamoDb/vue/tableView.vue

```diff
@@ -165,14 +186,22 @@ export default defineComponent({
         async executeQuery() {
             let sortKeyElement = document.getElementById('sortKey')
             let partitionKeyElement = document.getElementById('partitionKey')
-
+            let sortKeyValue = ''
+            if (sortKeyElement) {
+                sortKeyValue = (sortKeyElement as any).value
+            }
             const queryRequest = {
                 partitionKey: (partitionKeyElement as any).value,
-                sortKey: (sortKeyElement as any).value,
+                sortKey: sortKeyValue,
             }
             this.updatePageNumber()
             this.dynamoDbTableData = await client.queryData(queryRequest)
         },
+
+        showContextMenu(event: any, row: any) {
```

Comment:
Is this a precursor to https://github.com/aws/aws-toolkit-vscode/pull/5422?

Reply:
Yes